runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -946,6 +946,10 @@ private static void translateFn(
       Map<Long, TupleTag<?>> outputMap) {
 
     DoFnSignature signature = DoFnSignatures.getSignature(fn.getClass());
+    if (signature.processElement().isSplittable()) {
+      throw new UnsupportedOperationException(
+          String.format("FlinkRunner does not currently support splittable DoFn: %s", fn));
```

Comment:
This is the `DataflowRunner`. Using identifier interpolation would have caught this :-)